pkg/kubectl/resource/builder.go

```diff
@@ -342,28 +342,29 @@ func (b *Builder) ResourceTypeOrNameArgs(allowEmptySelector bool, args ...string
 			}
 		}
 		return b
+	} else {
```

Comment:
the function `ResourceTypeOrNameArgs`  set value `b.resourceTuples` or `b.resource` then return, it's an either or logic I thinks using `if else` will be more logical clearly